src/libcollectdclient/client.c

```diff
@@ -24,6 +24,14 @@
  *   Florian octo Forster <octo at collectd.org>
  **/
 
+#ifdef WIN32
+#include "gnulib_config.h"
+#include <winsock2.h>
+#else
+#include <sys/socket.h>
+#include <sys/un.h>
```

Comment:
moving these includes up will probably break some platform like OpenBSD. See 34471c08182f3ea9f977d8cea1e8934e59e01ef4 for an example.

Reply:
Ah ok. Moved them down. gnulib_config.h needs to be before config.h, so I left the Windows includes at the top.